README.md

```diff
@@ -195,14 +195,21 @@ Update browser settings:
     - In the mobile app Settings page, tap "Search" within the "General" section. There should be an option titled "Add Search Engine" to select. It should prompt you to enter a title and search query url - use the following elements to fill out the form:
       - Title: "Whoogle"
       - URL: "https://\<your whoogle url\>/search?q=%s"
-  - Firefox (Android)
+  - Firefox <79.0.0 (Android)
     - Navigate to your app's url
     - Long-press on the search text field
     - Click the "Add Search Engine" menu item
       - Select a name and click ok
     - Click the 3 dot menu in the top right
-    - Navigate to the settings menu and select the "search" sub-menu
+    - Navigate to the settings menu and select the "Search" sub-menu
     - Select Whoogle and press "Set as default"
+  - Firefox >=79.0.0 (Android)
+    - Click the 3 dot menu in the top right
+    - Navigate to the settings menu and select the "Search" sub-menu
+    - Click "Add search engine"
+    - Select the 'Other' radio button
+      - Name: "Whoogle"
+      - Search string to use: "http[s]://\<your whoogle url\>/search?q=%s"
```

Comment:
Minor nitpick, but just for consistency the URL should be wrapped as a code block (easier to copy/paste that way).

```suggestion
      - Search string to use: `http[s]://\<your whoogle url\>/search?q=%s`
```